src/thor/bidirectional_astar.cc

```diff
@@ -113,7 +104,6 @@ bool BidirectionalAStar::ExpandForward(GraphReader& graphreader,
                                        const GraphId& node,
```

Comment:
all the changes made to this `ExpandForward` were made in `ExpandReverse` and also in the other 2 algorithms `timedep_forward` and `timedep_reverse`